### DISCUSSION TITLE
How to change external URLs to load draco_wasm_wrapper and process.wasm from inner hosts

### DISCUSSION THREAD

**Discussion by PoltP** [splinetool/react-spline#216] [Q&A]
I'm using your library to show Spline animation.

Now we have CORS issues for the following URLs:
https://www.gstatic.com/draco/versioned/decoders/1.5.2/draco_wasm_wrapper.js
https://www.gstatic.com/draco/versioned/decoders/1.5.2/draco_decoder.wasm
https://unpkg.com/@splinetool/modelling-wasm@1.9.37/build/process.wasm

I've tried VanillaJS way and used `setUIWasmUrl`  to check like this
```
const wasm = 'https://unpkg.com/@splinetool/modelling-wasm@1.9.36/build/process.wasm?test=1';

export const Test = React.memo(() => {
	return (
		<canvas
			ref={(canvas) => {
				if (canvas) {
					const app = new Application(canvas);
					app.setUIWasmUrl(wasm);
					app.load(scene);
				}
			}}
		/>
	);
});
```
but this method does not work.

Please help me solve the issue

**Comment by jeslage** [comment]
We have the same issue, we cannot load draco from external services due to gdpr compliance. Is there a solution for this?

**Comment by purple97** [comment]
We have the same issue

**Comment by jeslage** [comment]
@PoltP @purple97 we solved the issue. 
We currently use the `@splinetool/runtime` package to load the spline animation in our frontend, because we dont use react on that project. This package has an option to pass the `wasmPath`. So if you download the animation from spline you should get all related draco files which you can host locally then. Maybe the react version has the same prop. :) 
Hope this helps!

Here is our initialization:
```
const app = new Application(canvas, {
  wasmPath: '/assets/draco/'
});
```

**Reply by jeslage** [reply]
Without testing or fully reading the docs, i assume that you can do it like this with the react-spline package:
```
<Spline scene="/path/to/file.splinecode" wasmPath="/assets/draco/" />
```
